### PR TITLE
UITOOL-996/merge next in master

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

Merging Next in Master to get React 18 in the release 🎉 

### Potential Breaking Changes

React 18 is now mandatory to use Plasma

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -188,6 +188,452 @@
 
 
 
+## 47.9.1 (2022-11-04)
```
Comment (GermainBergeron) [inline CHANGELOG.md:191]: I would revert all the changes to the Changelog, a new change will be generated when you merge your PR

File: packages/mantine/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@coveord/plasma-mantine",
-    "version": "47.9.3",
+    "version": "47.9.1",
```
Comment (GermainBergeron) [inline packages/mantine/package.json:3]: We should keep the latest version in the package.json I think

**Review by GermainBergeron — APPROVED** [review]

**Comment by GermainBergeron** [comment]
Also make sure to change the message when you merge to include the `BREAKING CHANGE`

File: packages/mantine/package.json (hunk 2 above)
Comment (aiheon) [reply packages/mantine/package.json:3]: oops 🙈  yes indeed

**Pull request merged**